From ceec4b7cb6acaf9ed6741946a485a474c22d6edf Mon Sep 17 00:00:00 2001
From: Fedora Kernel Team <kernel-team@fedoraproject.org>
Date: Mon, 10 Jan 2022 08:44:35 -0300
Subject: [PATCH] block, elevator: default blk-mq to bfq

blk-mq do not have proper Kconfig selector, so change it here
---
 block/elevator.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/block/elevator.c b/block/elevator.c
index adee58e..4b21058 100644
--- a/block/elevator.c
+++ b/block/elevator.c
@@ -561,7 +561,7 @@ static inline bool elv_support_iosched(struct request_queue *q)
 }
 
 /*
- * For single queue devices, default to using mq-deadline. If we have multiple
+ * For single queue devices, default to using bfq. If we have multiple
  * queues or mq-deadline is not available, default to "none".
  */
 static struct elevator_type *elevator_get_default(struct request_queue *q)
@@ -573,7 +573,7 @@ static struct elevator_type *elevator_get_default(struct request_queue *q)
 	    !blk_mq_is_shared_tags(q->tag_set->flags))
 		return NULL;
 
-	return elevator_find_get("mq-deadline");
+	return elevator_find_get("bfq");
 }
 
 /*
-- 
2.39.2

